doc/source/user/quickstart.rst

```diff
@@ -485,24 +490,27 @@ and other Python sequences.
     8
     >>> a[2:5]
     array([ 8, 27, 64])
-    >>> a[:6:2] = -1000    # equivalent to a[0:6:2] = -1000; from start to position 6, exclusive, set every 2nd element to -1000
+    # equivalent to a[0:6:2] = 1000;
+    # from start to position 6, exclusive, set every 2nd element to 1000
+    >>> a[:6:2] = 1000
     >>> a
-    array([-1000,     1, -1000,    27, -1000,   125,   216,   343,   512,   729])
+    array([1000,    1, 1000,   27, 1000,  125,  216,  343,  512,  729])
     >>> a[ : :-1]                                 # reversed a
-    array([  729,   512,   343,   216,   125, -1000,    27, -1000,     1, -1000])
+    array([  729,  512,  343,  216,  125, 1000,   27, 1000,    1, 1000])
     >>> for i in a:
-    ...     print(i**(1/3.))
+    ...     print(np.abs(i)**(1/3.))
```

Comment:
```suggestion
    ...     print(np.cbrt(i))
```
no need for `abs`, you made it positive already

Reply:
ok